### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by EkShunyam** [practical-tutorials/project-based-learning#673] [General]
i loved this project esp for self taught ppl 
thanks for putting it together and keep it up 

can we convert some of them into follow along labs - for a guided experience

**Comment by AutoNateAI** [comment]
https://github.com/user-attachments/assets/32ddcd4b-90ec-4411-813e-26a3ed3a2868

**Comment by katelord744** [comment]
Project based learning enhances critical thinking, problem-solving, and real world application making education more engaging and practical.
